src/cryptography/hazmat/backends/openssl/decode_asn1.py

```diff
@@ -548,17 +556,17 @@ def _decode_crl_distribution_points(backend, cdps):
             else:
                 rns = cdp.distpoint.name.relativename
                 rnum = backend._lib.sk_X509_NAME_ENTRY_num(rns)
-                attributes = []
+                attributes = set()
```

Comment:
Sorry about the delay. I am okay with making a breaking change here if we implemented it incorrectly (which it appears we, or more specifically, I, did). I also agree that should be a separate PR. To make sure I understand this then, you have it all adding to a single set so that it behaves as much like an RDN as possible, correct?